server/index.js

```diff
@@ -1,11 +1,12 @@
 /* eslint-disable no-console */
 const express = require('express');
```

Comment:
Fantastic work, I honestly have no comments. This is great. 

If you want something anyways, I'd say clean up line 22-38. If you look back on the grocery list practice we did with Josh, his methods are super clean and look really good. Has to do with db attacks and stuff, so they say. Let me know if you want any more clarification on that, but overall stellar work. You're crushing it.